psycop/projects/scz_bp/model_training/estimator_steps/infer_numeric_standardisation.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+
+import pandas as pd
+from deepchecks.utils.type_inference import infer_categorical_features
+from numpy.typing import ArrayLike
+from sklearn.preprocessing import StandardScaler
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.task.model_step import ModelStep
+
+
+class InferNumericStandardScaler(StandardScaler):
```

Comment:
Is z-score normalisation of categorical values represented as int a problem? 
They'd still appear ordinal with constant spacing after normalisation, right?